src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.h

```diff
@@ -0,0 +1,34 @@
+#ifndef RSYSLOG_PLUGIN_H
+#define RSYSLOG_PLUGIN_H
+
+#include <string>
+#include <memory>
+#include "syslog_parser.h"
+#include "events.h"
+#include "logger.h"
+
+using namespace std;
```

Comment:
do not use usings in header files

Reply:
These header files are not for external clients use. Only used by rsyslog plugin.c. This could have been sucked into rsyslog plugin itself, but separated just for readability.

Hence it does not make a diff.